format/spec.md

```diff
@@ -48,12 +48,12 @@ In addition to row-level deletes, version 2 makes some requirements stricter for
 
 Version 3 of the Iceberg spec extends data types and existing metadata structures to add new capabilities:
 
-* New data types: nanosecond timestamp(tz), unknown
+* New data types: nanosecond timestamp(tz), variant, geometry, geography, and unknown
```

Comment:
Not really related to the other changes in this pr. We can get it in now though since we do need to fix it eventually